.github/workflows/emscripten.yml

```diff
@@ -23,8 +23,17 @@ jobs:
     - name: Configure Ninja
       uses: seanmiddleditch/gha-setup-ninja@master
 
+    - name: Configure Emscripten SDK version
+      if: success()
+      shell: bash
+      run: |
+        cd $GITHUB_WORKSPACE/BuildTools/Scripts/github_actions
+        ./emscripten_sdk.sh
+
     - name: Download and Configure Emscripten SDK
-      uses: mymindstorm/setup-emsdk@v9
+      uses: mymindstorm/setup-emsdk@master
```

Comment:
Is it possible to run this command from `emscripten_sdk.sh`?

Reply:
I don't know. I'm experimenting

Reply:
The goal here is to maximize the reuse of logic. We would need to the same for other submodules, so it is preferable to have us little copy-paste as possible.